File: utbot-spring-analyzer/src/main/java/analyzers/PropertiesAnalyzer.java
package analyzers;

import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;
import java.util.ArrayList;

public class PropertiesAnalyzer {

    private final String propertiesFilePath;

    public PropertiesAnalyzer(String propertiesFilePath) {
        this.propertiesFilePath = propertiesFilePath;
    }

    public ArrayList<String> readProperties() throws IOException {
        ArrayList<String> props = new ArrayList<>();

        BufferedReader reader = new BufferedReader(new FileReader(propertiesFilePath));
        String line = reader.readLine();
        while (line != null) {
            props.add(line);
            line = reader.readLine();
        }

        reader.close();

        return props;
    }
}

File: utbot-spring-analyzer/src/main/kotlin/analyzers/PropertiesAnalyzer.kt
package analyzers

import java.io.BufferedReader
import java.io.FileReader
import java.io.IOException

class PropertiesAnalyzer(private val propertiesFilePath: String) {
    fun readProperties(): ArrayList<String> {
        val props = ArrayList<String>()

        val reader = BufferedReader(FileReader(propertiesFilePath))
        var line = reader.readLine()
        while (line != null) {
            props.add(line)
            line = reader.readLine()
        }

        reader.close()

        return props
    }
}